docs/source/upcoming_release_notes/1086-Adding_classes_for_AT1K2.rst

```diff
@@ -0,0 +1,30 @@
+1086 Adding classes for AT1K2
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- Attenuator module now has classes for two blade attenuators
```

Comment:
```suggestion
- Adds a preliminary attenuator class AT1K2 and base classes for similar
   two-blade ladder attenuators designed by JJ X-ray.
```

I say "preliminary" because the motion axes don't exist yet as far as EPICS is concerned (i.e., there's no IOC)